build/emoji/emoji_gen.mjs

```diff
@@ -141,30 +143,88 @@ function genSkinVariations(emoji, index, nextOrder) {
         variation.category = emoji.category;
         variation.subcategory = emoji.subcategory;
         variation.skins = skins;
-        variation.source_index = index;
-        variation.sort_order = order++;
         return variation;
     });
 }
 
+function trimPropertiesFromEmoji(emoji, emojiKeys) {
+    if (emojiKeys.includes('non_qualified')) {
+        Reflect.deleteProperty(emoji, 'non_qualified');
+    }
+
+    if (emojiKeys.includes('docomo')) {
+        Reflect.deleteProperty(emoji, 'docomo');
+    }
+
+    if (emojiKeys.includes('au')) {
+        Reflect.deleteProperty(emoji, 'au');
+    }
+
+    if (emojiKeys.includes('softbank')) {
+        Reflect.deleteProperty(emoji, 'softbank');
+    }
+
+    if (emojiKeys.includes('google')) {
+        Reflect.deleteProperty(emoji, 'google');
+    }
+
+    if (emojiKeys.includes('sheet_x')) {
+        Reflect.deleteProperty(emoji, 'sheet_x');
+    }
+
+    if (emojiKeys.includes('sheet_y')) {
+        Reflect.deleteProperty(emoji, 'sheet_y');
+    }
+
+    if (emojiKeys.includes('added_in')) {
+        Reflect.deleteProperty(emoji, 'added_in');
+    }
+
+    if (emojiKeys.includes('has_img_apple')) {
+        Reflect.deleteProperty(emoji, 'has_img_apple');
+    }
+
+    if (emojiKeys.includes('has_img_google')) {
+        Reflect.deleteProperty(emoji, 'has_img_google');
+    }
+
+    if (emojiKeys.includes('has_img_twitter')) {
+        Reflect.deleteProperty(emoji, 'has_img_twitter');
+    }
+
+    if (emojiKeys.includes('has_img_facebook')) {
+        Reflect.deleteProperty(emoji, 'has_img_facebook');
+    }
+
+    if (emojiKeys.includes('source_index')) {
+        Reflect.deleteProperty(emoji, 'source_index');
+    }
+
+    if (emojiKeys.includes('sort_order')) {
+        Reflect.deleteProperty(emoji, 'sort_order');
+    }
```

Comment:
@Willyfrog Can we also remove `image` and `fileName` since it is lower case of unified and appended with png to end respectively